### ISSUE TITLE
3.00 released but not advertized?

### ISSUE THREAD

**Issue by sandrotosi** [transmission/transmission#1223]
Hello,
i saw 3.00 was release (hooray!!) but it doesnt look like it was advertized or anything?

* on gh it's just a tag, not a "release" as previous ones
* on https://transmissionbt.com/ (and https://transmissionbt.com/download/), 2.94 is the version that's currently advertised as latest

so i'm wondering if there's anything going on or just lack of time, and we can start pushing 3.00 to the linux distros (i speak for Debian).

Thanks for transmission!

**Comment by ChaosBlades** [comment]
It is out on FreeBSD but it does not install the service so you can't actually start it. Tried to install from pkg and build from source and I get the same thing, no service to start. I was on latest repo and had to roll back to quarterly to get 2.94. I have been in a holding pattern to see what happens.

The "Official" link for FreeBSD from the website.
https://www.freebsd.org/cgi/ports.cgi?query=transmission&stype=name&sektion=net-p2p

**Comment by mvanbaak** [comment]
> It is out on FreeBSD but it does not install the service so you can't actually start it. Tried to install from pkg and build from source and I get the same thing, no service to start. I was on latest repo and had to roll back to quarterly to get 2.94. I have been in a holding pattern to see what happens.
> 
> The "Official" link for FreeBSD from the website.
> https://www.freebsd.org/cgi/ports.cgi?query=transmission&stype=name&sektion=net-p2p

Are you sure you are installing the package ```transmission-daemon```? That's the one that installs the daemon with the service script. Just tested in a new jail and works fine here.

**Comment by mvanbaak** [comment]
Can an 'official' release be made please? Right now I have a very hard time to convince tracker owners to whitelist transmission 3.00 as not one considers this a stable release.

Thanks!

**Comment by ChaosBlades** [comment]
> 
> 
> > It is out on FreeBSD but it does not install the service so you can't actually start it. Tried to install from pkg and build from source and I get the same thing, no service to start. I was on latest repo and had to roll back to quarterly to get 2.94. I have been in a holding pattern to see what happens.
> > The "Official" link for FreeBSD from the website.
> > https://www.freebsd.org/cgi/ports.cgi?query=transmission&stype=name&sektion=net-p2p
> 
> Are you sure you are installing the package `transmission-daemon`? That's the one that installs the daemon with the service script. Just tested in a new jail and works fine here.

I figured out the problem! Thank you. 
So on 2.94 'pkg install transmission' installs the following...
        transmission: 2.94_4
        transmission-cli: 2.94_5
        transmission-daemon: 2.94_3
        transmission-gtk: 2.94_5
        transmission-web: 2.94_5

On 3.00 it only installs...
        transmission: 3.00
        transmission-cli: 3.00
        transmission-gtk: 3.00
        transmission-web: 3.00

Looks like someone forgot to include the daemon dependency.

**Comment by ChaosBlades** [comment]
So 'pkg install transmission' installs everything you need on 2.94 but with 3.00 installing 'transmission' installs everything but 'transmission-daemon' and installing the daemon after that removes the 'transmission' and 'transmission-cli' packages. 

When I was doing a pkg upgrade it was installing everything like with 2.94 but it was breaking the daemon. I had to pkg remove transmission and transmission-daemon then just install the daemon to get things working.

So anyone with transmission pre 3.00 that does a pkg upgrade will need to remove all transmission packages and just install the daemon.

Now all I need is a change log to see what has changed. Not seeing anything different at all. Transmission Remote GUI still works. Integrations with other apps still work. Transfer speeds seems the same. Not seeing anything new in the Web-UI.

**Comment by mvanbaak** [comment]
@ChaosBlades The FreeBSD port/package is not something the transmission project itself provides. Let's ask questions and/or report bugs for that on the freebsd-ports mailinglist or on IRC.

**Comment by sandrotosi** [comment]
@ChaosBlades please bring your Freebsd packaging questions to a separate forum (maybe freebsd support?) or at least a separate issue here: you kinda hijacked the issue i opened specifically to inquiry about transmission release, and you're not helping figuring that out. thanks.

**Comment by ckerr** [comment]
BLUF: 3.00 is coming soon but it's not finalized and people shouldn't be rolling out downstream packages until the website has formal downoads.

We've had a couple of false starts on the 3.00 release. As I understand it, the main issue we're waiting on right now is for the new app signing mechanism to be in place for Windows. @mikedld is working on this and there's a related PR for the website at https://github.com/transmission/transmission.github.io/pull/26 because the signing requires the website add a code signing policy.

We are probably going to re-tag yet again and re-roll the assets. While waiting on signing to get into place, I've been landing a handful of smaller / safer PRs whose authors have been waiting for awhile.

**Comment by thesamesam** [comment]
That's understandable. Please be aware that given the CVE was just disclosed, there will be a lot of people eager to update. That's how I ended up finding this thread.

Unfortunately the patch doesn't cleanly apply to the last release, so it cannot be applied downstream easily. If you foresee it being a while, a point release may be appreciated via the existing mechanisms.

**Comment by ckerr** [comment]
3.00 is now released _and_ advertised

**Comment by xavery** [comment]
@sandrotosi Just mentioning you explicitly in case you missed a notification - a Debian package update would be very welcome. :-) Thanks!
